docs/advisors/checks/replica_running_skipping_errors_or_idempotent_mode.md

```diff
@@ -0,0 +1,22 @@
+# The advisor checks for two replication specific configuration options. 
+
+
+## Description
+
+Normally, replication stops when an error occurs on the replica, which gives you the opportunity to resolve the inconsistency in the data manually. The variable  `[slave | replica]_skip_errors` causes the replication SQL thread to continue replication when a statement returns any of the errors listed in the variable value.
+
+The configuration `[slave | replica]_exec_mode` controls how a replication thread resolves conflicts and errors during replication. IDEMPOTENT mode causes suppression of duplicate-key and no-key-found errors; STRICT means no such suppression takes place. 
+This check ensures the check on recommended replication setup and suggests against using IDEMPOTENT mode or skipping any replication errors.
```

Comment:
controls the way a replication thread resolves either conflicts or errors during replication. The `STRICT` mode is the default value and does not suppress conflicts or errors. The `IDEMPOTENT` mode suppresses duplicate-key and key-not-found errors, and this mode should only be used if you are sure these errors can be ignored.

Reply:
@kedarvj pls check this one, as I have no idea )